designer/client/page-create.js

```diff
@@ -94,7 +93,9 @@ class PageCreate extends React.Component {
           <label className='govuk-label govuk-label--s' htmlFor='link-from'>Link from (optional)</label>
           <select className='govuk-select' id='link-from' name='from'>
             <option />
-            {pages.map(page => (<option key={page.path} value={page.path}>{page.path}</option>))}
+            {pages.map(page => (<option key={page.path} value={page.path}>{page.title}</option>)).sort(function (a, b) {
```

Comment:
cleaner to do the sort outside of this. is this sorting on the array of elements? if you do sort on pages you can control what its being sorted by (page.path, page.title etc)

you can use fat arrow instead of `function` and forego the return. `.sort( ( a, b ) => a.props.children > b.props.children ? 1 : -1)`